beep/tests/test_structure.py

```diff
@@ -592,6 +595,20 @@ def test_from_raw_cycler_run_arbin(self):
         self.assertEqual(pcycler_run.barcode, "EL151000429559")
         self.assertEqual(pcycler_run.protocol, r"2017-12-04_tests\20170630-4_65C_69per_6C.sdu")
 
+        all_summary = pcycler_run.summary
+        reg_dyptes = all_summary.dtypes.tolist()
```

Comment:
Is reg_dyptes a typo of reg_dtypes, or does it have some other meaning?

Reply:
That's so weird. I must have had a typo in the first one and then used autocomplete for all of the rest. Should be fixed now